climpred/prediction.py

```diff
@@ -261,6 +272,12 @@ def compute_hindcast(
                 }
             )
         # broadcast dims when apply over member
+
+        # Had to rechunk to put time all in 1 chunk,
+        # also testing mmember all in one chunk?
+        # Runs really slow even though I am testing with a subsetted region
```

Comment:
Either delete here or add the chunking back in. I can't remember where I said this but I think you could do `dask.is_dask_collection()` and if true, chunk time to -1. Then you don't impose chunks when they were in memory but can handle the time issue.